app/src/main/java/org/simple/clinic/storage/migrations/Migration_78.kt

```diff
@@ -0,0 +1,18 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("Classname")
+class Migration_78 @Inject constructor() : Migration(77, 78) {
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.inTransaction {
+      execSQL("""
+        ALTER TABLE "TeleconsultRecord" ADD COLUMN "syncStatus" TEXT NOT NULL
```

Comment:
Do we need to set a default value for existing records? This will set the value to existing records to `null` which will cause a crash if we try to read it.